samples/java-protobuf-eventsourced-customer-registry-subscriber/docker-compose-integration.yml

```diff
@@ -0,0 +1,42 @@
+# this docker-compose is used by the integration tests. To run it, we must first publish
+# java-spring-eventsourced-customer-registry image locally. To do so, run the following command from the root of the project:
+# mvn clean install docker:build
+
+version: "3"
+services:
+  kalix-proxy:
+    image: gcr.io/kalix-public/kalix-proxy:1.1.9-M2
```

Comment:
That's not a problem, especially for this sample that is used by use for testing. 

It could be a problem if we used a not yet deployed proxy version in the templates, but the sample is ok.